Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1821,22 +1848,5 @@ public Task SendMessageToUserDeviceSessions<T>(string deviceId, string name, T d
 
             return SendMessageToSessions(sessions, name, data, cancellationToken);
         }
-
-        public Task SendMessageToUserDeviceAndAdminSessions<T>(string deviceId, string name, T data, CancellationToken cancellationToken)
-        {
-            CheckDisposed();
```

Comment:
are you sure we still need `CheckDisposed` if you remove this?

Reply:
Yes